web/components/issues/description-form.tsx

```diff
@@ -81,7 +90,7 @@ export const IssueDescriptionForm: FC<IssueDetailsProps> = (props) => {
     });
   }, [issue, reset]);
 
-  const debouncedTitleSave = useDebouncedCallback(async () => {
+  const debouncedFormSave = useDebouncedCallback(async () => {
```

Comment:
for debounce don't use an external package to do it. You can create a hook that implements debounce for you.

Reply:
I'll fix this in a later PR @sriramveeraghanta, don't want to make this PR longer 🙌